moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -619,9 +614,6 @@ bool ServoCalcs::internalServoUpdate(Eigen::ArrayXd& delta_theta,
   // Set internal joint state from original
   internal_joint_state_ = original_joint_state_;
 
-  // Enforce SRDF Velocity, Acceleration limits
-  delta_theta = enforceVelocityLimits(joint_model_group_, parameters_->publish_period, delta_theta);
```

Comment:
This is moved down so that velocity scaling doesn't happen so soon. Previously the order was:

- apply velocity limits
- apply velocity scaling for collisions
- low-pass filter
- apply position limits

The new order is:

- apply velocity scaling for collisions
- low-pass filter
- apply velocity limits
- apply position limits

Reply:
The low pass filter in a way is a velocity clamp too right?  It smooths large changes in position.  Have you measured how much velocity is being reduced by the low pass filter?

Reply:
Are the commanded values for each of these steps published somewhere so we can plot them and measure how much each step is changing the result for a given input?

Reply:
These comments don't really seem to apply to this PR.

Yes, I've plotted the filter output many times in the past and it even has some unit tests.